stetho/src/main/java/com/facebook/stetho/inspector/database/ContentProviderDatabaseDriver.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.inspector.database;
+
+import android.content.ContentResolver;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteException;
+import com.facebook.stetho.inspector.protocol.module.Database;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@ThreadSafe
+public class ContentProviderDatabaseDriver extends Database.DatabaseDriver {
+
+  private final static String sDatabaseName = "content-providers";
+
+  private final ContentProviderSchema[] mContentProviderSchemas;
```

Comment:
There is null checks later in code. But if we want to prevent this from being a null, then throwing runtime exception from constructor can work I think.